src/lib.js

```diff
@@ -47,11 +51,76 @@ export async function getLitContracts(wallet) {
  */
 export async function encrypt(filePath, accessControlConditions, chain = 'ethereum') {
   const litClient = await getLit()
+  const fileContent = await fs.promises.readFile(filePath)
+  const blob = new Blob([fileContent])
+  return await encryptFile({ file: blob, accessControlConditions, chain }, litClient)
+}
+
+/**
+ *
+ * @param {string} filePath
+ * @param {import('@lit-protocol/types').AccessControlConditions} accessControlConditions
+ */
+export async function encryptLargeFile(filePath, accessControlConditions) {
+  const stat = await fs.promises.stat(filePath)
 
-  const fileContent = await fs.readFile(filePath)
-  let blob = new Blob([fileContent])
+  // Generate a random symmetric key and initialization vector
```

Comment:
where is this taken from? just want to make sure we're following well established patterns for any encryption we're doing